qsimcirq/qsim_circuit.py

```diff
@@ -15,6 +15,82 @@
 import numpy as np
 
 import cirq
+from qsimcirq import qsim
+
+
+def _cirq_gate_kind(gate):
+  if isinstance(gate, cirq.ops.identity.IdentityGate):
+    if gate.num_qubits == 1:
+      return qsim.kI
+    return qsim.kI2
```

Comment:
Check if `if gate.num_qubits == 2` for `qsim.kI2`.

Reply:
Done.